README.md

```diff
@@ -28,7 +28,7 @@ In addition to taking over the functionality of Instagram, the functioning of Pi
 It is possible to import your data from Instagram.
 
 
-**Shipped version:** 0.12.4-glitch.1.9.1~ynh1
+**Shipped version:** 1.9.1~ynh1
```

Comment:
This looses all link to Pixelfed original version, I'm not a big fan.

Reply:
The question is rather, how can I set up the version that will not fail the tests?

Reply:
can you try using a + ? We only use minus because of docker's incompatibility with semver, and it might be why yunohost considers it as an unstable version.

That would give you : 0.12.4+glitch.1.9.1~ynh1

Reply:
> The question is rather, how can I set up the version that will not fail the tests?

Does it fail any test ? I don't see it :thinking:

Reply:
@ericgaspar could you please check now ?